common/lib/common-utils/src/fluidErrorDemos.ts

```diff
@@ -0,0 +1,119 @@
+/*!
```

Comment:
Is this file supposed to be deleted before merging?

Reply:
yep. This PR is not at all to merge (it's across multiple build dependencies anyway), it's a conversation piece.